build/charts/antrea/templates/controller/rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: antrea-read-secrets
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: secret-reader
+subjects:
+  - kind: ServiceAccount
+    name: antrea-controller
+    namespace: {{ .Release.Namespace }}
```

Comment:
By this definition, we expect that the authSecret is under namespace kube-system? If so, we need to at least add some comments for this. Otherwise, controller will get RBAC problem if the namespace defined in the SupportBundleCollection is not kube-system.

Reply:
User should modify the namespace for role `antrea-read-secrets` according to his own setup. This is a sample for antrea. 

I would like to describe this in a future patch dedicated for documentation.